valeriyapetrova/hometask-2/get_middle_character.rb

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+# begin resault # = gets.chomp
+
+# get middle from world
+class Middle
+  puts 'Enter a string:'
+  def initialize(user_input = gets.chomp)
```

Comment:
let's move user_input setting in a separate method (gets.chomp as a default param in initialized doesn't look good)